### PR TITLE
miniz: fix violations of coding rules

### PR DESCRIPTION
**Pull request by sunghan-chang**
no spaces at the start of line

Signed-off-by: sunghan-chang <sh924.chang@samusng.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [18b9e065d5eff4ee263363fac260fd3c0a8ae9fe] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [18b9e065d5eff4ee263363fac260fd3c0a8ae9fe] - Code Rule Check Result:
os/include/tinyara/miniz/miniz.h:43: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:44: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:45: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:46: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:47: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:48: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:69: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:97: ERROR: [IDT_M_TAB] code indent should use tabs where possible

**Comment by seinfra** [comment]
Target : [1fb3dc9c826287afbf7b7388387dcd24c5dbdf21] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1fb3dc9c826287afbf7b7388387dcd24c5dbdf21] - Code Rule Check Result:
os/include/tinyara/miniz/miniz.h:43: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:44: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:45: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:46: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:47: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:48: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:69: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/miniz/miniz.h:97: ERROR: [IDT_M_TAB] code indent should use tabs where possible

File: os/include/tinyara/miniz/miniz.h [hunk 1]
```diff
@@ -651,85 +643,81 @@ typedef mz_bool (*tdefl_put_buf_func_ptr)(const void *pBuf, int len, void *pUser
...
-{
-    TDEFL_MAX_HUFF_TABLES = 3,
-    TDEFL_MAX_HUFF_SYMBOLS_0 = 288,
-    TDEFL_MAX_HUFF_SYMBOLS_1 = 32,
-    TDEFL_MAX_HUFF_SYMBOLS_2 = 19,
-    TDEFL_LZ_DICT_SIZE = 32768,
-    TDEFL_LZ_DICT_SIZE_MASK = TDEFL_LZ_DICT_SIZE - 1,
-    TDEFL_MIN_MATCH_LEN = 3,
-    TDEFL_MAX_MATCH_LEN = 258
+enum {
```
Comment (btheosam) [inline os/include/tinyara/miniz/miniz.h]: Two enumeration has the same value 3. Is it okay?

File: os/include/tinyara/miniz/miniz.h [hunk 2]
```diff
@@ -651,85 +643,81 @@ typedef mz_bool (*tdefl_put_buf_func_ptr)(const void *pBuf, int len, void *pUser
...
-    TDEFL_LZ_CODE_BUF_SIZE = 64 * 1024,
-    TDEFL_OUT_BUF_SIZE = (TDEFL_LZ_CODE_BUF_SIZE * 13) / 10,
-    TDEFL_MAX_HUFF_SYMBOLS = 288,
-    TDEFL_LZ_HASH_BITS = 15,
-    TDEFL_LEVEL1_HASH_SIZE_MASK = 4095,
-    TDEFL_LZ_HASH_SHIFT = (TDEFL_LZ_HASH_BITS + 2) / 3,
-    TDEFL_LZ_HASH_SIZE = 1 << TDEFL_LZ_HASH_BITS
+enum {
+	TDEFL_LZ_CODE_BUF_SIZE = 64 * 1024,
+	TDEFL_OUT_BUF_SIZE = (TDEFL_LZ_CODE_BUF_SIZE * 13) / 10,
```
Comment (btheosam) [inline os/include/tinyara/miniz/miniz.h]: Enums are integar values. is it guaranteed to be integer?

**Pull request merged**